flink-runtime/src/main/java/org/apache/flink/runtime/metrics/groups/TaskIOMetricGroup.java

```diff
@@ -71,6 +75,7 @@ public class TaskIOMetricGroup extends ProxyMetricGroup<TaskMetricGroup> {
 
     public TaskIOMetricGroup(TaskMetricGroup parent) {
         super(parent);
+        this.taskStartTime = System.currentTimeMillis();
```

Comment:
The timestamp of creating this metric group does not equals to the timestamp the task start processing. The difference in between sometimes can be quite large, e.g., restoring a large state.

I think we should add a trigger in this metric group and pull it in `StreamTask#invoke`, right before `runMailboxLoop()`.